front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/TextQuestion/Analysis/AnalysisInsights.tsx

```diff
@@ -23,7 +23,13 @@ import messages from '../../../messages';
 import Question from './Question';
 import Summary from './Summary';
 
-const AnalysisInsights = ({ analysis }: { analysis: IAnalysisData }) => {
+const AnalysisInsights = ({
+  analysis,
+  hasOtherResponses,
```

Comment:
The easiest way I could figure out to do this, is by passing `hasOtherResponses` around everywhere. Is there a more elegant solution?

Reply:
Since it's only 2 "layers" deep, I think this is ok :)

Reply:
In my opinion, when it starts to be more than 3 layers deep, then I typically refer to it as "prop drilling" and it should have another solution :)